### DISCUSSION TITLE
Pin tab

### DISCUSSION THREAD

**Discussion by talestomaz** [Zettlr/Zettlr#3726] [Ideas]
I'm slowly getting used to Zettlr and trying to find a suitable workflow. For this, it seems that, among other things, I'd need to be able to pin some tabs (e.g., my to-do-list, the parent note that links others, my most relevant current project, etc.). Is there a way to do it? If not, could this be implemented? I searched here and didn't find anything related here in the discussions. Thanks!

**Comment by nathanlesage** [comment]
Pinned tabs are already implemented, though only on the develop branch. So they'll come with the next release. If you're ready to accept some risk, feel free to test out the [nightly releases](https://nightly.zettlr.com/).
